proportionality.qmd

```diff
@@ -13,50 +13,46 @@ The draft currently has no official status. It is a work in progress and is subj
 
 # Proportionality
 
-All analysis shall be assured, and the assurance should be proportionate to the potential effect it will have, size and complexity of the analysis. The level of assurance should be guided by a structured assessment of the business risks. 
+All analysis shall be assured. The assurance should be proportionate to the potential effect it will have and the size and complexity of the analysis. The level of assurance should be guided by a structured assessment of the business risks. 
 
 The assurer and the analyst shall be independent. The degree of separation depends on many factors including the importance of the output, and the size and complexity of the analysis. This does not mean that the analyst should not undertake assurance, rather that there shall also be some formal independent assurance. 
 
-## Introduction
+## Factors for determining appropriate assurance
 
-Think about and deliver appropriate (proportionate) levels of assurance for your analysis.  There is a need to be confident in analysis delivered, but there is no point spending months assuring simple analysis that will have a minor influence on a decision.  
+While there is a need to be confident in the analysis it is not necessary to spend months assuring simple analysis that will have a minor influence on a decision.  The level of analysis should be appropriate (proportionate) to the analysis.
```

Comment:
Should be "the level of assurance should be appropriate (proportionate)... " in line 22